### PR TITLE
save wip

### PR DESCRIPTION
**Pull request by jkdev601**

### CODE REVIEW

File: src/renderer/components/ui/SentFundsModal.js [hunk 1]
```diff
@@ -103,7 +104,7 @@ export const SentFundsModal = ({
           >
             <Grid container className={classes.root}>
               <Grid className={classes.title} item xs={12}>
-                <Typography variant='h3'>Sent funds</Typography>
+                <Typography variant='h3'>{title || `Sent Fundss`}</Typography>
```
Comment (NorbertBodziony) [inline src/renderer/components/ui/SentFundsModal.js:107]: Typo

File: src/renderer/components/ui/sendMessageSeparate/SendMessageSeparateInitial.js [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+              </Typography>
+            }
+          />)}
+        />
+      </Grid>
+      {(shouldSendAnonymously || (isUserSelected && values.sendAnonymously)) && (
+        <Grid container item>
+          <Grid className={classes.infoBox} item>
+            <Typography className={classes.typo} variant='body2'><span className={classes.bold}>Warning:</span>{` anonymity in Zcash has limits. Lorem ipsum where we explain this
+            anonymity in Zcash has limits. Lorem ipsum where we explain this,
```
Comment (NorbertBodziony) [inline src/renderer/components/ui/sendMessageSeparate/SendMessageSeparateInitial.js:233]: Lorem ipsum ? Or just placeholder?

File: src/renderer/components/ui/sendMessageSeparate/SendMessageSeparateMain.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+import * as R from 'ramda'
+import { Formik } from 'formik'
+import { withStyles } from '@material-ui/core/styles'
+import { AutoSizer } from 'react-virtualized'
+import { Scrollbars } from 'react-custom-scrollbars'
+import * as Yup from 'yup'
+import BigNumber from 'bignumber.js'
+
+import { MESSAGE_SIZE } from '../../../zbay/transit'
+// import { createTransfer } from '../../../zbay/messages'
```
Comment (NorbertBodziony) [inline src/renderer/components/ui/sendMessageSeparate/SendMessageSeparateMain.js:12]: leftover

File: src/renderer/components/ui/sendMessageSeparate/SendMessageSeparateInitial.js (hunk 2 above)
Comment (holmesworcester) [reply src/renderer/components/ui/sendMessageSeparate/SendMessageSeparateInitial.js:233]: The text should be:

Warning: anonymity in Zcash has limits. For example, an attacker may be able to discover your IP address or link this message to your other activity on Zbay.

File: src/renderer/components/ui/sendMoney/SendMoneyForm.js [hunk 4]
```diff
@@ -342,12 +342,13 @@ export const SendMoneyForm = ({
         <TextField
           name='memo'
           placeholder={
-            values.recipient.length === 35
+            values.recipient ? values.recipient.length === 35
               ? `You can't include message to transparent address`
```
Comment (holmesworcester) [inline src/renderer/components/ui/sendMoney/SendMoneyForm.js:346]: text should be:

Only shielded Zcash addresses (beginning with a 'z') can receive messages

**Pull request merged**